bin/git-ignore-io

```diff
@@ -0,0 +1,124 @@
+#/bin/sh
+
+gitignore_io_url="https://www.gitignore.io/api/"
+
+default_path="$HOME/.gi_list"
+gi_list=`cat $default_path | tr "," "\n"`
+
+update_gi_list() {
+    curl -L -s  "${gitignore_io_url}/list" > ~/.gi_list
+}
+
+print_in_alphabetical_order() {
+    for i in {a..z};
+    do
+        echo "$gi_list" | grep "^$i" | tr "\n" " "
```

Comment:
As I see, the default order of "https://www.gitignore.io/api/list" is in alphabetical.
IMHO, even if we don't believe in the default order, sorting the items in `"$gi_list"` is better.